### PR TITLE
[#2291]:+ fixed inconsistent plugin placement.

### PR DESCRIPTION
**Pull request by StevenGuapaBV**
Moved all the plugins to have Plugins folder as root. Fixed folder structure for plugins to be placed in a root Plugin folder. Also removed duplicate plugins in adminhtml di which were already present in etc/di

### Description
To be consistent in the location where plugins are kept I updated the module CatalogUrlRewrite where a some model plugins where not in the Plugins root folder. 

### Fixed Issues (if relevant)
1. magento/magento2#2291: Inconsistent location for plugins

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Review by miguelbalparda — APPROVED** [review]

File: app/code/Magento/Customer/view/frontend/templates/widget/name.phtml [hunk 1]
```diff
@@ -33,7 +33,6 @@ $suffix = $block->showSuffix();
     </label>
     <div class="control">
         <fieldset class="fieldset fieldset-fullname">
-        <div class="fields">
```
Comment (sidolov) [inline app/code/Magento/Customer/view/frontend/templates/widget/name.phtml]: Looks like this changes are not related to the PR. Please, revert

**Review by sidolov — CHANGES_REQUESTED** [review]

**Comment by StevenGuapaBV** [comment]
Hi @sidolov,

I reverted the change.

**Review by sidolov — APPROVED** [review]

**Comment by sidolov** [comment]
Hi @StevenGuapaBV , unit tests place should be consistent with plugins place,  please, move tests according to new plugin namespace

**Comment by sidolov** [comment]
Hi @StevenGuapaBV , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**